python/packages/ai/teams/ai/prompts/completion_config.py

```diff
@@ -57,6 +57,8 @@ class CompletionConfig:
 
         top_p (float): The model's top_p as a number between 0 and 2.
           Defaults to 0.
+
+        data_sources (Optional[list[object]]): List of data sources to ground the answer in.
```

Comment:
are you able to utilize the `data_source` class type for this?

Reply:
Sorry if I misunderstood you. I believe the `DataSource` class is used to describe a data source in prompt, not to configure the OpenAI API request. https://github.com/microsoft/teams-ai/blob/main/python/packages/ai/teams/ai/data_sources/data_source.py#L17